src/Http/Controllers/CategoryController.php

```diff
@@ -13,6 +15,9 @@ public function show(int $categoryId)
         config(['frontend.subcategories' => $category->subcategories->pluck('name', 'entity_id')]);
         session(['latest_category_path' => $category->path]);
 
-        return view('rapidez::category.overview', compact('category'));
+        $response = response()->view('rapidez::category.overview', compact('category'));
+        $response->setCache(['etag' => md5($response->getContent() ?? ''), 'last_modified' => $category->updated_at]);
+
+        return $response;
```

Comment:
Can we write this cleaner like this?
```suggestion
        return response()
            ->view('rapidez::category.overview', compact('category'));
            ->setCache([
                'etag' => md5($response->getContent() ?? ''),
                'last_modified' => $category->updated_at
            ]);
```

Or maybe even better:

```suggestion
        return response()
            ->view('rapidez::category.overview', compact('category'));
            ->setEtag(md5($response->getContent()))
            ->setLastModified($category->updated_at);
```

Reply:
I can call the setEtag and setLastModified directly instead of using setCache.
Unfortunately due to the `setEtag` requiring the response content i cannot chain it onto the response()->viev()

Reply:
Ah I see, maybe this:

```suggestion
        $response = response()->view('rapidez::category.overview', compact('category'));

        return $response
            ->setEtag(md5($response->getContent()))
            ->setLastModified($category->updated_at);
```

Reply:
But it's pretty repeated, maybe we can extract it somehow?